website/docs/r/opsworks_ecs_cluster_layer.html.markdown

```diff
@@ -0,0 +1,68 @@
+---
+subcategory: "OpsWorks"
+layout: "aws"
+page_title: "AWS: aws_opsworks_ecs_cluster_layer"
+description: |-
+  Provides an OpsWorks HAProxy layer resource.
+---
+
+# Resource: aws_opsworks_ecs_cluster_layer
+
+Provides an OpsWorks ECS Cluster layer resource.
+
+## Example Usage
+
+```hcl
+resource "aws_opsworks_ecs_cluster_layer" "lb" {
+  stack_id       = aws_opsworks_stack.main.id
+  stats_password = "foobarbaz"
```

Comment:
@DrFaust92 shouldn't this be `ecs_cluster_arn` ?

Reply:
fixed